helpers/handleReceiverChainContractDeploy.ts

```diff
@@ -0,0 +1,18 @@
+import { deployContract } from './deployContract'
+
+export interface handleReceiverChainContractDeployConfig {
+  drawCalculator: string,
+  drawBuffer: string,
+  prizeDistributionBuffer: string,
+}
+
+
+export async function handleReceiverChainContractDeploy(deploy: Function, deployer: string, config: handleReceiverChainContractDeployConfig = {
+  drawCalculator: '',
+  drawBuffer: '',
+  prizeDistributionBuffer: ''
+}) {
+  await deployContract(deploy, 'L2TimelockTrigger', deployer, [deployer, config.drawBuffer, config.prizeDistributionBuffer, config.drawCalculator])
+}
+
+export default handleReceiverChainContractDeploy
```

Comment:
again...this file is only called in one place.  Just put the function there!  Doesn't need a file + type + separate declaration

Reply:
It's called in the mumbai deploy script now too.